src/dynamic.rs

```diff
@@ -116,11 +116,10 @@ impl Dynamic {
         Ok(())
     }
 
-    pub fn sync(&self, linked: Option<&mut SectionContent>, eh: &Header) -> Result<(), Error> {
+    pub fn sync(&self, linked: Option<&mut SectionContent>, _: &Header) -> Result<(), Error> {
         match self.content {
-            DynamicContent::String(ref s) => match linked {
-                Some(&mut SectionContent::Strtab(ref mut strtab)) => {
-                    let off = strtab.insert(s.bytes().collect()) as u32;
```

Comment:
this doesn't look right. you removed the actual function body

Reply:
The function body is not doing anything. The case is still active and is not doing anything, as before.

Reply:
strtab.insert does do something, i'm confused why you're saying it doesn't.

Reply:
Fuck...